great_expectations/data_context/data_context.py

```diff
@@ -1082,43 +1082,12 @@ def get_parameters_in_evaluation_parameter_store_by_run_id(self, run_id):
             None
         """
         if self.evaluation_parameter_store.has_key(run_id):
-            return self.evaluation_parameter_store.get(run_id)
+            return copy.deepcopy(
```

Comment:
Is this still needed?

Reply:
Not sure, tbh.

I know I misread the way `bind`ing worked, but without this copy, the potential to do the same thing is still there. I think we're better off keeping it, just in case. Not deeply committed to that opinion, though...